packages/@aws-cdk/cloudformation-include/lib/cfn-include.ts

```diff
@@ -231,6 +293,39 @@ export class CfnInclude extends core.CfnElement {
     this.conditions[conditionName] = cfnCondition;
   }
 
+  private createNestedStack(nestedStackId: string, props?: CfnIncludeProps): void {
+    if (nestedStackId in this.template.Resources) {
+      if (this.template.Resources[nestedStackId].Type === 'AWS::CloudFormation::Stack' && props?.nestedStacks) {
+        const nestedStackScope = new core.NestedStack(this, 'Nested' + nestedStackId, {
+          parameters: this.template.Resources[nestedStackId].Properties.Parameters,
+          notificationArns: this.template.Resources[nestedStackId].Properties.NotificationArns,
+          timeout: this.template.Resources[nestedStackId].Properties.Timeout,
+        });
+
+        if (nestedStackScope.nestedStackResource) {
```

Comment:
Actually, don't do this `if`. Force the non-null case with `!`. (`nestedStackScope. nestedStackResource!`).

If this is ever `undefined`, that is a serious bug, and we should fail-fast here.